src/Section.js

```diff
@@ -16,17 +18,34 @@ export class Section extends React.Component {
 
         return (
             <div class="section flex-column">
-                <Header {...headerInfo} />                        
-                {
-                    cards && cards.map(card => ( 
-                        <Card 
-                            key={card._id}                            
-                            onDeleteCard={onDeleteCard} 
-                            onModalInfo={onModalInfo}
-                            {...card} 
-                        />
-                    ))
-                }
+                <Header {...headerInfo} />  
+                <Droppable droppableId={status}>
+                    {(provided) => (
+                        <div
+                            ref={provided.innerRef}>
+                            {cards && cards.map((card, index) => (
+                                <Draggable 
+                                    key={card._id}
+                                    draggableId={card._id} 
+                                    index={index}>
+                                    {(provided) => (
+                                        <div
+                                            ref={provided.innerRef}
+                                            {...provided.draggableProps}
+                                            {...provided.dragHandleProps}>                                            
+                                            <Card 
+                                                key={card._id}                            
```

Comment:
do you need key here? I think you needed since previously rendered through array of cards